packages/nodes-base/nodes/Slack/MessageDescription.ts

```diff
@@ -603,10 +603,12 @@ export const messageFields: INodeProperties[] = [
 							{
 								name: 'Actions',
 								value: 'actions',
+								description: 'i18n TODO DELETE Some description',
```

Comment:
here are some TODOs in the file, leftovers?

Reply:
These were introduced on purpose and will be removed. i18n can only translate strings that exist, and I needed to test if a param with this type, level of nesting, etc. had a particular property, so this was the fastest. All can be identified with `i18n TODO DELETE`